service/src/main/java/bio/terra/cbas/runsets/monitoring/SmartRunsPoller.java

```diff
@@ -207,7 +208,7 @@ private Run updateDatabaseRunStatus(
                         updatableRun.recordId(),
                         updatableRun.runId(),
                         updatableRun.engineId(),
-                        e.getMessage());
+                        WdsClientUtils.extractErrorMessage(e.getMessage()));
```

Comment:
I'm not sure if there's anywhere else in the CBAS codebase you'd want to use this. I don't see any other test failures, but perhaps there are exception cases that don't have test coverage.